LayoutGrid/LayoutGrid.jsx

```diff
@@ -0,0 +1,78 @@
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+
+class LayoutGrid extends MaterialComponent {
+	constructor() {
+		super();
+		this.componentName = "layout-grid";
+	}
+	materialDom(props) {
+		return (
+			<div className="mdc-layout-grid" ref={control=>{this.control = control;}} {...props}>
+				{props.children}
+			</div>
+		);
+	}
+}
+
+
+/**
+ * @prop columns = 0
+ * @prop desktop = 0
+ * @prop tablet = 0
+ * @prop phone = 0
+ * @prop order = 0
+ * @prop align = ''
+ */
+class LayoutGridCell extends MaterialComponent {
+	constructor() {
+		super();
+		this.componentName = "layout-grid__cell";
+	}
+	createClassName(props) {
+		const baseClass = "mdc-layout-grid__cell--";
+		const classes = [];
+
+		if (props.columns) {
+			classes.push(baseClass + "span-" + props.columns) ;
+		}
+
+		if (props.desktop) {
```

Comment:
shall we move desktop, tablet, phone to a friendlier prop name?
desktopSpan, mobileSpan, tabletSpan may be?